types/event_bus.go

```diff
@@ -90,51 +91,62 @@ func (b *EventBus) UnsubscribeAll(ctx context.Context, subscriber string) error
 func (b *EventBus) Publish(eventType string, eventData TMEventData) error {
 	// no explicit deadline for publishing events
 	ctx := context.Background()
-	b.pubsub.PublishWithTags(ctx, eventData, map[string]string{EventTypeKey: eventType})
+	_ = b.pubsub.PublishWithEvents(ctx, eventData, map[string][]string{EventTypeKey: {eventType}})
```

Comment:
Huh should we really be ignoring the err here ?

Reply:
We were before! I simply added a blank identifier for linting reasons. But since this function signature returns an error already, I'll handle it and return it to the caller.